src/equations/compressible_mhd_3d.jl

```diff
@@ -0,0 +1,185 @@
+@doc raw"""
+    CompressibleMhdDiffusion3D(gamma, inv_gamma_minus_one,
+                               μ, Pr, eta, kappa,
+                               equations, gradient_variables)
+
+These equations contain the viscous Navier-Stokes equations coupled to
+the magnetic field together with the magnetic diffusion applied
+to mass, momenta, magnetic field and total energy together with the advective terms from
+the [`IdealGlmMhdEquations3D`](@ref).
+Together they constitute the compressible, viscous and resistive MHD equations with energy.
+
+- `gamma`: adiabatic constant,
+- `mu`: dynamic viscosity,
+- `Pr`: Prandtl number,
+- `eta`: magnetic diffusion (resistivity)
+- `equations`: instance of the [`IdealGlmMhdEquations3D`](@ref)
+- `gradient_variables`: which variables the gradients are taken with respect to.
+Defaults to `GradientVariablesPrimitive()`.
+
+Fluid properties such as the dynamic viscosity $\mu$ and magnetic diffusion $\eta$
+can be provided in any consistent unit system, e.g.,
+[$\mu$] = kg m⁻¹ s⁻¹.
+
```

Comment:
This docstring is incomplete. It should include the form of the additional visco-resistive fluxes. Also maybe point to one of the papers from Andrés for the particular form and scaling of the governing equations.

Reply:
Added equations in the docs and a reference to Andre's paper.